reporter/src/funTest/assets/npm-test-with-exclude-scan-results.yml

```diff
@@ -21,6 +21,7 @@ analyzer:
   end_time: "1970-01-01T00:00:00Z"
   environment:
     ort_version: "6fd0fde"
+    java_version: "1.8.0_181"
```

Comment:
Shouldn't that be `<REPLACE_JAVA>`? Also several times below.

Reply:
No, but I have to admit I got confused by this myself. The logic is: In expected result files we use the `<REPLACE_...` placeholders and use `patchExpectedResult` in the test. This file, however, is not an expected result file, but an actual result file, used as the *input* for the test. These files should contain (close to) real data as generated by a scan, and not contain any placeholders, but be modified by `patchActualResult`.